compose-post.sh

```diff
@@ -30,6 +30,13 @@ set -x
 cp -f /usr/lib/locale/locale-archive /usr/lib/locale/locale-archive.tmpl
 build-locale-archive
 
+# https://github.com/openshift/os/issues/96
+# sudo group https://github.com/openshift/os/issues/96
+echo '%sudo        ALL=(ALL)       NOPASSWD: ALL' > /etc/sudoers.d/coreos-sudo-group
+# Also set up sudo access for core user for now...but
+# we may require users to explicitly ask for this.
+echo 'core ALL=(ALL) NOPASSWD: ALL' > /etc/sudoers.d/coreos-core-user
```

Comment:
Since right now, our hack for adding the core user is in `cloud.ks`, we might as well keep this part there as well, e.g.:

```diff
-user --name=core --groups='wheel,adm,systemd-journal'
+user --name=core --groups='wheel,adm,systemd-journal,sudo'
```

?

Reply:
Good idea!  Updated :arrow_down: